Formula/erlang@18.rb

```diff
@@ -0,0 +1,119 @@
+# Major releases of erlang should typically start out as separate formula in
+# Homebrew-versions, and only be merged to master when things like couchdb and
+# elixir are compatible.
+class ErlangAT18 < Formula
+  desc "Programming language for highly scalable real-time systems"
+  homepage "https://www.erlang.org/"
+  head "https://github.com/erlang/otp.git"
+
+  stable do
```

Comment:
Please remove the `stable do` block and place `url` and `sha256` above where `head` is currently.